tests/system/test_integration.py

```diff
@@ -2,27 +2,12 @@
 import unittest
 
 from apmserver import ElasticTest, ExpvarBaseTest
-from apmserver import ClientSideBaseTest, SmapIndexBaseTest, SmapCacheBaseTest
+from apmserver import ClientSideBaseTest, SmapIndexBaseTest, SmapCacheBaseTest, SplitIndicesTest
 from beat.beat import INTEGRATION_TESTS
 
 
 class Test(ElasticTest):
 
-    @unittest.skipUnless(INTEGRATION_TESTS, "integration test")
-    def test_onboarding_doc(self):
```

Comment:
I liked having the explicit `test_onboarding_doc`, and not test it implicit within the `test_load_docs_with_template_and_add_transaction`.